.vscode/settings.json

```diff
@@ -8,7 +8,7 @@
   "eslint.validate": [
     "javascript",
     "javascriptreact",
-    { "language": "typescript", "autoFix": true }
+    "typescript"
```

Comment:
I suspect the changes in this file should be reverted, since this file applies to the whole repo?

Reply:
Or if you think this change should be applied to the whole repo, it will need to be reviewed by someone from the JS team.

Reply:
Sorry about that. You're right. Will revert this.